src/whatsnew/1.4.rst

```diff
@@ -48,7 +48,7 @@ Version 1.4.0
   updates and deletion of databases. :commit:`917d8988`
 * :issue:`1772`: Prevent invalid JSON output when using `all_or_nothing`
   :ref:`of bulk API <api/db/bulk_docs>`. :commit:`dfd39d57`
-* Add a :config:option:`configurable whitelist <couch_httpd_auth/public_fields>`
+* Add a :config:option:`configurable whitelist <chttpd_auth/public_fields>`
```

Comment:
We don't want to modify whatsnew/1.4 since the release is already out

Reply:
It is a link reference, and if I don't change it, it will fail the test.

Reply:
Hmm, in that case we should turn the link into a regular string perhaps?